### ISSUE TITLE
Ventoy not creating 2 partitions

### ISSUE THREAD

**Issue by averageandroidconsumer** [ventoy/Ventoy#2581]
### Official FAQ

- [x] I have checked the official FAQ.

### Ventoy Version

1.0.95

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

No. I didn't try these alternative boot modes.

### BIOS Mode

UEFI Mode

### Partition Style

GPT

### Disk Capacity

16GB

### Disk Manufacturer

SanDisk

### Image file checksum (if applicable)

No.

### What happened?

so basically ventoy is supposed to create 2 partitions on a usb drive,the efi partition and the larger exfat partition
but there is only one
im asking because when booting from the drive a security violation error appeared,i knew i had to enroll the key but i can't find the key

**Comment by averageandroidconsumer** [comment]
it randomly worked again
